### PR TITLE
Feat/Add canonical ID lookup (KB-313)

### PR DESCRIPTION
**Pull request by notrueblood**

### CODE REVIEW

File: cofactr/graph.py [hunk 1]
```diff
@@ -247,11 +247,12 @@ def get_products(
 
         extracted_producs = res.json()
```
Comment (josephsayad) [inline cofactr/graph.py:248]: Nit: Correct typo `extracted_producs` to `extracted_products`

File: cofactr/graph.py [hunk 2]
```diff
@@ -247,11 +247,12 @@ def get_products(
...
 
-        Product = schema_to_product[schema]  # pylint: disable=invalid-name
+        if schema != ProductSchemaName.INTERNAL:
+            Product = schema_to_product[schema]  # pylint: disable=invalid-name
 
-        extracted_producs["data"] = [
-            Product(**data) for data in extracted_producs["data"]
-        ]
+            extracted_producs["data"] = [
```
Comment (josephsayad) [inline cofactr/graph.py:253]: Do we want to skip parsing if `schema == ProductSchemaName.INTERNAL`?

Comment (notrueblood) [reply cofactr/graph.py:253]: Yeah; we want to parse if a special, non-internal, schema is used

File: cofactr/graph.py (hunk 1 above)
Comment (notrueblood) [reply cofactr/graph.py:248]: Thanks

**Pull request merged**